great_expectations/core/validation.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from great_expectations._docs_decorators import public_api
+from great_expectations.compatibility.pydantic import BaseModel
+
+if TYPE_CHECKING:
+    from great_expectations.core.batch_config import BatchConfig
+    from great_expectations.core.expectation_suite import ExpectationSuite
+    from great_expectations.datasource.fluent.interfaces import DataAsset
+
+
+class Validation(BaseModel):
+    """
+    Responsible for running a suite against data and returning a validation result.
+
+    Args:
+        name: The name of the validation.
+        data: An asset or batch config to validate.
+        suite: A grouping of expectations to validate against the data.
+
+    """
+
+    name: str
+    data: DataAsset | BatchConfig
+    suite: ExpectationSuite
```

Comment:
If I make a change to a suite, how can I ensure that changes cascade through our persistence layer?

Reply:
Do you mean a user has access to both a validation and a suite. They update the suite, how do we guarantee they see that in the validation?

Would be make use of a property and have it do an external call each time someone gets this property?